packages/p2p-v2/src/routes/AppContent/index.scss

```diff
@@ -8,6 +8,10 @@
             @include mobile {
                 width: calc(100vw / 4);
             }
+
+            & > span {
+                font-size: 1.4rem;
```

Comment:
can we use `Text` component?

Reply:
For this i'm targeting the Tabs component which doesn't have any props for textSize, so I had to do this to reduce the sizing, I can bring this issue with @shayan-deriv